src/index.js

```diff
@@ -23,6 +23,7 @@ Hypergrid.behaviors = require('./behaviors');
 Hypergrid.dataModels = require('./dataModels');
 Hypergrid.features = require('./features');
 Hypergrid.analytics = require('./Shared.js').analytics;
+Hypergrid.DataSourceBase = require('fin-hypergrid-data-source-base');
```

Comment:
Let's make this `Hypergrid.dataSources = require('./dataSources');` for consistency and also in case we every decide to add another data source file. **Please create ./dataSources/index.js.**